graveler/committed/meta_range_manager.go

```diff
@@ -10,9 +10,16 @@ import (
 )
 
 type Params struct {
-	// ApproximateRangeSizeBytes is a target for sizes of range partitions.  It should be
-	// kept fixed to maximize re-use of range partitions.
-	ApproximateRangeSizeBytes uint64
+	// MinRangeSizeBytes is the smallest size for splitting a range partition as a result
+	// of adding a record.  Smaller ranges are still possible due to re-using an existing
+	MinRangeSizeBytes uint64
+	// MaxRangeSizeBytes is the largest size of a range partition.  In practice the range
+	// is split only after an additional record.
+	MaxRangeSizeBytes uint64
+	// RangeSizeRaggedness allows raggedness in splitting range partitions.  It is
+	// the expected number of records after MinRangeSizeBytes at which to split the range
+	// -- ranges are split at the first key with hash divisible by this raggedness.
+	RangeSizeRaggedness float64
```

Comment:
Other two are in bytes so I would mention the difference in the name:
```suggestion
	RangeSizeEntriesRaggedness float64
```